### PR TITLE
Fix Issue 15281 - Add std.experimental.allocator to the build script

### PR DESCRIPTION
**Pull request by MetaLang**
There were a few lines that used spaces instead of tabs, so I changed those to tabs as well. These changes seem to work, but please review them carefully as I don't actually know make; I just imitated what was already there. I also have not tested the `win64.mak` changes, will try to do that tomorrow as it's already quite late.

### CODE REVIEW

File: win32.mak [hunk 1]
```diff
@@ -145,11 +145,32 @@ SRC_STD_LOGGER= std\experimental\logger\core.d std\experimental\logger\filelogge
 	std\experimental\logger\multilogger.d std\experimental\logger\nulllogger.d \
 	std\experimental\logger\package.d
 
+SRC_STD_ALLOC_BB= std\experimental\allocator\building_blocks\affix_allocator.d \
+	std\experimental\allocator\building_blocks\allocator_list.d \
+	std\experimental\allocator\building_blocks\bucketizer.d \
```
Comment (aG0aep6G) [inline win32.mak]: Missed bitmapped_block.d? Same in win64.mak.

File: win32.mak [hunk 2]
```diff
@@ -145,11 +145,32 @@ SRC_STD_LOGGER= std\experimental\logger\core.d std\experimental\logger\filelogge
...
+	std\experimental\allocator\building_blocks\free_list.d \
+	std\experimental\allocator\building_blocks\free_tree.d \
+	std\experimental\allocator\building_blocks\kernighan_ritchie.d \
+	std\experimental\allocator\building_blocks\null_allocator.d \
+	std\experimental\allocator\building_blocks\quantizer.d \
+	std\experimental\allocator\building_blocks\region.d \
+	std\experimental\allocator\building_blocks\scoped_allocator.d \
+	std\experimental\allocator\building_blocks\segregator.d \
+	std\experimental\allocator\building_blocks\stats_collector.d \
+	std\experimental\allocator\building_blocks\package.d
```
Comment (aG0aep6G) [inline win32.mak]: I think package.d would better be sorted alphabetically, too, i.e. between null_allocator.d and quantizer.d. Same in win64.mak.

File: win32.mak [hunk 3]
```diff
@@ -145,11 +145,32 @@ SRC_STD_LOGGER= std\experimental\logger\core.d std\experimental\logger\filelogge
...
+
+SRC_STD_ALLOC= std\experimental\allocator\common.d std\experimental\allocator\gc_allocator.d \
+	std\experimental\allocator\mallocator.d std\experimental\allocator\mmap_allocator.d \
+	std\experimental\allocator\showcase.d std\experimental\allocator\typed.d \
+	std\experimental\allocator\package.d \
```
Comment (aG0aep6G) [inline win32.mak]: ditto alphabetic order

**Comment by JackStouffer** [comment]
This should have been opened on stable.

File: win32.mak (hunk 2 above)
Comment (MetaLang) [inline win32.mak]: I just copied what was done for the other modules. I don't see any reason to change it.

**Comment by MetaLang** [comment]
I opened a new PR against stable: #3843

**Pull request closed without merging**